datafusion/physical-expr/src/aggregate.rs

```diff
@@ -91,6 +91,9 @@ impl AggregateExprBuilder {
         }
     }
 
+    /// Constructs an `AggregateFunctionExpr` from the builder
+    ///
+    /// Note that an [`Self::alias`] must be provided before calling this method.
```

Comment:
It would also be nice to update the error message below 

```rust
            None => return internal_err!("alias should  provided"),
```

Perhaps to something like

```rust
            None => return internal_err!("AggregateExprBuilder::alias must be provided prior to build"),
```